tests/helper_tests.py

```diff
@@ -135,7 +135,8 @@ def test_dbref_to(self, get_factory, create):
         get_factory.return_value = factory
         create.return_value = {"_id": 1234}
         func = dbref_to("user")
-        self.assertEqual({"$id": 1234, "$ref": "users"}, func())
+        dbref = func()
+        self.assertEqual({"$id": 1234, "$ref": "users"}, {'$id': dbref.id, '$ref': dbref.collection})
```

Comment:
I'd change this assertion to be against an actual DBRef, rather than converting/comparing it to a dictionary